<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [ErrorValidationSuite]" guice-stage="DEVELOPMENT" verbose="0">
  <parameter name="fooPass" value="fooPass"/>
  <parameter name="fooEmail" value="fooEmail"/>
  <parameter name="lockedEmail" value="locked_out_user"/>
  <parameter name="pass" value="secret_sauce"/>
  <listeners>
    <listener class-name="testComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="WrongCredentialsErrorValidation(failed)" verbose="0">
    <classes>
      <class name="tests.ErrorValidationTest">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="takeScreenShotOfFailedTest"/>
        </methods>
      </class> <!-- tests.ErrorValidationTest -->
    </classes>
  </test> <!-- WrongCredentialsErrorValidation(failed) -->
</suite> <!-- Failed suite [ErrorValidationSuite] -->
